apps/account-api/src/controllers/v1/handles-v1.controller.ts

```diff
@@ -103,7 +116,7 @@ export class HandlesControllerV1 {
     try {
       const expiration = await this.handlesService.getExpiration();
       const payload = { baseHandle: newHandle, expiration };
-      const encodedPayload = u8aToHex(u8aWrapBytes(this.handlesService.encodePayload(payload).toU8a()));
+      const encodedPayload = u8aToHex(this.handlesService.encodePayload(payload).toU8a());
```

Comment:
My only questions here are:
1. Was this working before, and if so, why? Or is is just that we can now simplify the client signing code on the other side?
2. Is this confirmed working as expected now after this change?

Reply:
My understanding is that it depended on how the payload was signed.
1- If the payload was signed directly with the keypair then it works as expected since there is no mandetory wrapping for that
2- If the payload was signed by the polkadot wallet as raw data then the wallet would wrap it in Bytes regardless. So Having it already wrapped would cause issues.

I think to b e able to support both cases we probably would like to go the unwrapped route since in option 1 we can always wrap it in bytes before signing.

Reply:
@mattheworris Anything that I might be missing?

Reply:
Suggestion: can we add e2e tests for these payloads?

Reply:
We're going to have to do a whole series of tickets for E2E tests, after the initial round of hardening tickets are done, I think.